optaplanner-examples/src/main/java/org/optaplanner/examples/common/experimental/Sequence.java

```diff
@@ -41,10 +45,32 @@ protected Sequence(ConsecutiveSetTree<ValueType_, ?, ?> sourceTree, NavigableSet
         this.count = count;
     }
 
-    public NavigableSet<ValueType_> getItems() {
+    public Collection<ValueType_> getItems() {
```

Comment:
Users will only use this for iteration. They do not need to know what sort of `Collection` it is. GIve your future self some freedom - don't make promises you don't need to.

Reply:
On the other hand, maybe in this case, it's good that we tell people that the `Sequence` items are, indeed, in a sequence. :-) I leave this up to you.